src/adapter/threads.ts

```diff
@@ -92,6 +92,7 @@ export class ExecutionContext {
 
 export type Script = {
   url: string;
+  urlRegexp?: string;
```

Comment:
This should not be a property of Scripts

Reply:
Removed